code/src/components/GameStart.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+
+import { game } from '../reducers/game'
+import { generateStartMove } from '../reducers/fetch'
+import { Button, StyledButton } from '../lib/Button'
+import styled from 'styled-components'
+
+const StartButton = styled(StyledButton)`
+  margin-top: 20px;
+  width: 100px;
+`
+export const GameStart = () => {
+  const dispatch = useDispatch()
+  const [inputValue,  setInputValue] = useState("")
+
+  const username = useSelector((store) => store.game.username);
```

Comment:
Since you are not using semicolons in the rest of your project, this one should be removed. 🥳